builtin/providers/aws/structure_test.go

```diff
@@ -824,18 +824,29 @@ func TestFlattenResourceRecords(t *testing.T) {
 			Value: aws.String("127.0.0.1"),
 		},
 		&route53.ResourceRecord{
-			Value: aws.String("127.0.0.3"),
+			Value: aws.String("\"abc def\" \"xxx yyy\""),
 		},
 	}
 
-	result := flattenResourceRecords(expanded)
+	expected := []string{
+		"127.0.0.1",
+		"abc def\" \"xxx yyy",
+	}
+
+	result := flattenResourceRecords(expanded, "TXT")
```

Comment:
I think we should also test at least 1 other record type which will remain as is (i.e. verify that we're not removing quotes in such case and won't in the future).

Would you mind adding such test?